modules/serverless-sinkbinding-kn.adoc

```diff
@@ -3,37 +3,32 @@
 // serverless/knstive_eventing/serverless-sinkbinding.adoc
 
 [id="serverless-sinkbinding-kn_{context}"]
-= Using SinkBinding with the Knative CLI (`kn`)
+= Creating a sink binding by using the Knative CLI
```

Comment:
Since all of these are being changed from SinkBinding -> sink binding, should the _topic_map.yml file also be updated?